armi/reactor/parameters/parameterDefinitions.py

```diff
@@ -152,6 +152,10 @@ class Serializer:
         their version. It is also good practice, whenever possible, to support reading
         old versions so that database files written by old versions can still be read.
 
+    .. impl:: Implements ability to use a custom DB serializer for parameters.
```

Comment:
Okay, this is our first time adding `impl` tags, so there is no way you should have known this.

But lets create a good standard for other people.

* Let's not start with the word "Implements".
* Maybe just: "Custom parameter serializer"